lib/stdlib/test/io_SUITE.erl

```diff
@@ -210,6 +210,10 @@ float_w(Config) when is_list(Config) ->
 
     ok.
 
+calling_self(Config) when is_list(Config) ->
```

Comment:
There is a testcase in this SUITE called `error_info` that should also be updated.